src/lib/util/Config.ts

```diff
@@ -54,7 +54,16 @@ const DEFAULT_CONFIG: ConfigContent = {
 				"!**/bower_components/**"
 			],
 			disabled: false
-        }
+        },
+		{
+			name: ENGINE.RETIRE_JS,
+			targetPatterns: [
+				'**/*.js',
+				'!**/node_modules/**',
+				'!**/bower_components/**'
+			],
+			disabled: false
```

Comment:
All `2.x` releases are going to be based off the `release` branch, right? If that's the case, then I think we can leave this engine enabled by default.

Reply:
2.x will be based off of `dev`. I think it would be best to disable this by default.

Reply:
Cool. Then we should also wait to merge this until we fix the bug that includes disabled engines in the output of `scanner:rule:list` and `scanner:rule:describe`, right?